playbooks/roles/ocp-config/tasks/chrony.yaml

```diff
@@ -1,17 +1,52 @@
-- name: Create temporary chrony.conf file
-  template:
-    src: chrony.conf.j2
-    dest: /tmp/chrony.conf.tmp
+- name: Install chrony on bastion
+  yum:
+    name: chrony
+    state: present
 
-- name: slurp contents of temporary chrony.conf file
-  slurp:
-    src: /tmp/chrony.conf.tmp
-  register: chronybase64
+- name: Open NTP port on firewall
+  firewalld:
+      service: ntp
+      permanent: yes
+      immediate: yes
+      state: enabled
 
-- name: Generate Chrony machineconfig
-  template:
-    src: chrony-machineconfig.j2
-    dest: "{{ workdir }}/manifests/99-{{item}}-chrony-configuration.yaml"
-  loop: 
-    - master
-    - worker
+- name: Configure chrony to synchronize with ntp servers
+  lineinfile:
+    path: /etc/chrony.conf
+    regexp: '^server {{ item.server }} '
```

Comment:
Do we want to comment out the default pool/servers lines?  If yes then what we do when chronyconfig.content.servers list is empty? 

```
pool 2.rhel.pool.ntp.org iburst
```

Reply:
Ok, I think I can remove it only if we have something to add as servers...

Reply:
Make sense.

Reply:
This way the setup even works just by toggling the enabled flag.

Reply:
done